tool/speed.cc

```diff
@@ -169,8 +169,8 @@ struct free_functor {
   }
 };
 
-#if defined(OPENSSL_WINDOWS)
-#define AllocAligned malloc
+#if defined(OPENSSL_WINDOWS) || defined(OPENSSL_ANDROID)
```

Comment:
@bstrong Do you think this is needed? I commented out OPENSSL_ANDROID in cmake file and it's still working.